tox.ini

```diff
@@ -24,6 +24,9 @@ commands = pre-commit run --all-files --show-diff-on-failure
 [flake8]
 max-line-length = 88
 exclude = .eggs,.tox
+# rationale here:
+# https://github.com/psf/black/blob/master/docs/the_black_code_style.md#slices
+extend-ignore = E203
```

Comment:
ping @ssbarnea

Reply:
I already approved but apparently I do not get enough rights for the vote to matter (is gray, not green).